rom/tests/runRegressionTests.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 
 #SBATCH -N 1
-#SBATCH -t 1:00:00
-#SBATCH -p pdebug
+#SBATCH --exclusive
+#SBATCH -t 2:00:00
```

Comment:
There was a very slow test that took about 1 hour, but @kevinhkhuynh removed it, and now the tests all run in under 1 hour. The time limit `-t 1:00:00` could be restored here.

Reply:
Restored.